webui/react/src/pages/TrialDetails/TrialActions.tsx

```diff
@@ -43,11 +44,20 @@ const TrialActions: React.FC<Props> = ({ trial, onClick, onSettled }: Props) =>
     setButtonStates(state => ({ ...state, tensorboard: false }));
   }, [ trial.id, onSettled ]);
 
+  let trialCompletedCheckpointSum = 0;
+  experiment.trials.map((trial) => {
+    trialCompletedCheckpointSum += trial.numCompletedCheckpoints;
+  });
+
   const actionButtons: ConditionalButton<TrialDetails>[] = [
     {
-      button: <Button
-        key={Action.Continue}
-        onClick={onClick(Action.Continue)}>Continue Trial</Button>,
+      button: (trialCompletedCheckpointSum > 0 ? (
+        <Button key={Action.Continue} onClick={onClick(Action.Continue)}>Continue Trial</Button>
+      ) : (
+        <Tooltip key={Action.Continue} title={'No checkpoints found. Cannot continue trial.'}>
+          <Button disabled={true}>Continue Trial</Button>
```

Comment:
**nit:** I think you can just have `disabled` without the explicit `={true}` defined, but doesn't hurt to be explicit.